<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>List</class>
 <widget class="QWidget" name="List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>290</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#background{
	background-color: rgba(121, 122, 122, 228);
	border-radius: 20px;
	
}
QPushButton:hover{
color: rgb(66, 67, 67)
}
QPushButton{
	background-color: rgba(255, 255, 255, 0);
font-size: 16px;
}
QPushButton#star{
	image: url(:/images/icons/star.svg);
}
</string>
  </property>
  <widget class="QLabel" name="header">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>7</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>18</pointsize>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Ulubione miasta</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>30</y>
     <width>291</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Kraków</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Warszawa</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="text">
       <string>Gdańsk</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Gliwice</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_6">
      <property name="text">
       <string>Poznań</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>290</width>
     <height>270</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="star">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>7</y>
     <width>71</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>header</zorder>
  <zorder>verticalLayoutWidget</zorder>
  <zorder>star</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
